### ISSUE TITLE
Build issue on RedHat 6

### ISSUE THREAD

**Issue by jaudibert-euronext** [iuscommunity-pkg/git2u#6]
Hi,

We have the following issue when we build git2u on RedHat 6.
For the record, build of the v1.12 works fine

```
...
Processing files: git2u-2.13.0-1.ius.el6.x86_64
Provides: git = 2.13.0-1.ius.el6 git(x86-64) = 2.13.0-1.ius.el6
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(PartialHardlinkSets) <= 4.0.4-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1 rpmlib(VersionedDependencies) <= 3.0.3-1
Requires: /bin/sh /usr/bin/perl libc.so.6()(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3)(64bit) libc.so.6(GLIBC_2.3.4)(64bit) libc.so.6(GLIBC_2.4)(64bit) libc.so.6(GLIBC_2.7)(64bit) libpcre.so.0()(64bit) libpthread.so.0()(64bit) libpthread.so.0(GLIBC_2.2.5)(64bit) libpthread.so.0(GLIBC_2.3.2)(64bit) librt.so.1()(64bit) librt.so.1(GLIBC_2.2.5)(64bit) libz.so.1()(64bit) libz.so.1(ZLIB_1.2.0)(64bit) perl >= 0:5.008 perl(File::Basename) perl(File::Find) perl(File::Spec) perl(Getopt::Long) perl(Git) perl(Git::I18N) perl(lib) perl(strict) perl(warnings) rtld(GNU_HASH)
Conflicts: git < 2.13.0
Obsoletes: git2u-bzr <= 2.6.4-2.ius git2u-hg <= 2.6.4-2.ius
Processing files: git2u-all-2.13.0-1.ius.el6.noarch
Processing files: git2u-core-2.13.0-1.ius.el6.x86_64
error: File must begin with "/": GPLv2
error: File must begin with "/": COPYING


RPM build errors:
    File must begin with "/": GPLv2
    File must begin with "/": COPYING

```

**Comment by carlwgeorge** [comment]
TLDR: Install epel-rpm-macros from the EPEL repository.

That is happening because your build system doesn't understand the `%license` macro we are using.  That macro was introduced in [rpm 4.11](http://rpm.org/wiki/Releases/4.11.0#package-building), so it is built in to EL7, but not EL6.  In order to do the right thing on all RHEL releases, the recommended action is to use the `%license` macro but fallback to `%doc` macro when necessary.  In the past, we did this with a compatibility fallback ([suggested by Fedora](https://pagure.io/packaging-committee/issue/411#comment-147363)) in every spec file.

`%{!?_licensedir:%global license %%doc}`

More recently, EPEL (which is already required for IUS) introduced a package called epel-rpm-macros, which includes a [system-wide compatibility fallback](https://src.fedoraproject.org/cgit/rpms/epel-rpm-macros.git/tree/macros.zzz-epel?h=el6&id=5093e6edda9cc2e01820f1ea9cd320973e9caab3#n19).  That package is part of the buildsys-build group, so it should be installed on any rpm development machine with EPEL enabled (`yum install @buildsys-build`).  Since that now exists, we have started removing the compatibility fallback within our individual spec files.
